Adapt docs to PennyLane theme (#64)
PennyLaneAI/pennylane-forest: README.rst

.. header-start-inclusion-marker-do-not-remove

> Contains the PennyLane Forest plugin. This plugin allows different Rigetti-pyQuil devices to work with

Reviewer: There seeem to be many terms used for Rigetti software/devices:

Forest:
> PyQuil, along with quilc, the QVM, and other libraries, make up what is called the Forest SDK.

PyQuil:
> PyQuil is a Python library for quantum programming using Quil, the quantum instruction language developed at Rigetti Computing.

Maybe just good to leave this as is?
```suggestion
Contains the PennyLane Forest plugin. This plugin allows different Rigetti devices to work with
```

PennyLane --- the wavefunction simulator, the Quantum Virtual Machine (QVM), and Quantum Processing
Units (QPUs).

`pyQuil <https://pyquil.readthedocs.io>`_ is a Python library for quantum programming using the
quantum instruction language (Quil) --- resulting quantum programs can be executed using the
`Rigetti Forest SDK <https://pyquil-docs.rigetti.com/en/stable/>`_ and the `Rigetti QCS
<https://qcs.rigetti.com/>`_.

`PennyLane <https://pennylane.readthedocs.io>`__ is a cross-platform Python library for quantum machine

> learning, automatic differentiation, and optimization of hybrid quantum-classical computations.

Reviewer: We have an updated description on the main site now:

> PennyLane is a cross-platform Python library for differentiable programming of quantum computers. Train a quantum computer the same way as a neural network.

Maybe worth adjusting this to include "differentiable programming of quantum computers"?

Installation
============

Installation of PennyLane-Forest, as well as all required Python packages mentioned above, can be installed via ``pip``:
::

   	$ python -m pip install pennylane-forest

Make sure you are using the Python 3 version of pip.

Alternatively, you can install PennyLane-Forest from the source code by navigating to the top directory and running
::

	$ python setup.py install

Dependencies
~~~~~~~~~~~~

.. highlight:: bash

PennyLane-Forest requires the following libraries be installed:

* `Python <http://python.org/>`_ >=3.6

as well as the following Python packages:

* `PennyLane <http://pennylane.readthedocs.io/>`_

> * `pyQuil <http://docs.rigetti.com/>`_

Reviewer 2: Might be better to link directly to the pyQuil docs:
![image](https://user-images.githubusercontent.com/6934626/98719489-d193ff00-235d-11eb-9886-5e57d73658ed.png)

```suggestion
* `pyQuil <https://pyquil-docs.rigetti.com/en/stable/>`_
```

> PennyLane-Forest requires the following libraries be installed:
> 
> * `Python <http://python.org/>`_ >=3.6
> 
> as well as the following Python packages:
> 
> * `PennyLane <http://pennylane.readthedocs.io/>`_
> * `pyQuil <https://pyquil-docs.rigetti.com/en/stable/>`_
> 
> If you currently do not have Python 3 installed, we recommend
> `Anaconda for Python 3 <https://www.anaconda.com/download/>`_, a distributed version
> of Python packaged for scientific computation.

Reviewer: Maybe worth moving the sentence on Anaconda up to the Python requirement?
```suggestion
PennyLane-Forest requires the following libraries be installed:

* `Python <http://python.org/>`_ >=3.6

If you currently do not have Python 3 installed, we recommend
`Anaconda for Python 3 <https://www.anaconda.com/download/>`_, a distributed version
of Python packaged for scientific computation.

The following Python packages are also required:

* `PennyLane <http://pennylane.readthedocs.io/>`_
* `pyQuil <https://pyquil-docs.rigetti.com/en/stable/>`_
```

Additionally, if you would like to compile the quantum instruction language (Quil) and run it
locally using a quantum virtual machine (QVM) server, you will need to download and install the
Forest software development kit (SDK):

* `Forest SDK <https://pyquil-docs.rigetti.com/en/stable/>`_

Alternatively, you may sign up for Rigetti's Quantum Cloud Services (QCS) to acquire a Quantum Machine
Image (QMI) which will allow you to compile your quantum code and run on real quantum processing units (QPUs),
or on a preinstalled QVM. Note that this requires a valid QCS account.

> * `Quantum Cloud Services <https://www.rigetti.com/>`_

Reviewer 2: Here, perhaps the Rigetti docs could be nice to link to. Since the first heading is "Quantum Cloud Services Documentation".
```suggestion
* `Quantum Cloud Services <https://docs.rigetti.com/en/>`_
```

Installation
============

PennyLane-Forest, as well as all required Python packages mentioned above, can be installed via ``pip``:
::

   	$ python -m pip install pennylane-forest

Make sure you are using the Python 3 version of pip.

> Alternatively, you can install PennyLane-Forest from the source code by navigating to the top-LEVEL directory and running

Reviewer 3: :thinking: 
```suggestion
Alternatively, you can install PennyLane-Forest from the source code by navigating to the top-level directory and running
```
